test/Dialect/FIRRTL/lowerXMR.mlir

```diff
@@ -0,0 +1,116 @@
+// RUN: circt-opt %s  --firrtl-lower-xmr -split-input-file |  FileCheck %s
+
+// CHECK-LABEL: firrtl.circuit "xmr"
+firrtl.circuit "xmr" {
+  firrtl.module private @Test(out %x: !firrtl.ref<uint<2>>) {
+    %w = firrtl.wire : !firrtl.uint<2>
+    firrtl.ref.send %x, %w : !firrtl.ref<uint<2>>
+  }
+  firrtl.module @xmr() {
+    %test_x = firrtl.instance test @Test(out x: !firrtl.ref<uint<2>>)
+    %x = firrtl.ref.resolve %test_x : !firrtl.ref<uint<2>>
+    // This should be lowered to:
+    //  %0 = firrtl.verbatim.expr "{{0}}.{{1}}" : () -> !firrtl.uint<2>
+    //  {symbols = [#hw.innerNameRef<@xmr::@xmr_sym>, #hw.innerNameRef<@Test::@xmr_sym>]}
+  }
+  // CHECK:   firrtl.module private @Test() {
+  // CHECK:     %w = firrtl.wire sym @xmr_sym   : !firrtl.uint<2>
+  // CHECK:   }
+  // CHECK:   firrtl.module @xmr() {
+  // CHECK:     firrtl.instance test sym @xmr_sym  @Test()
+  // CHECK:     %0 = firrtl.verbatim.expr 
+  // CHECK-SAME: > !firrtl.uint<2> {symbols = [#hw.innerNameRef<@xmr::@xmr_sym>, #hw.innerNameRef<@Test::@xmr_sym>]}
```

Comment:
nit: 
```suggestion
  // CHECK-SAME: {symbols = [#hw.innerNameRef<@xmr::@xmr_sym>, #hw.innerNameRef<@Test::@xmr_sym>]}
```

This can also use `CHECK-LITERAL` and check the `{{0}}.{{1}}` part.